components/apps_form/apps_form_component.tsx

```diff
@@ -159,6 +159,7 @@ export class AppsForm extends React.PureComponent<Props, State> {
 
         this.setState({fieldErrors});
         if (Object.keys(fieldErrors).length !== 0) {
+            this.setState({formError: 'Please fix all field errors'});
```

Comment:
does this get localized?

Reply:
It can be.  I added localization for the error.

Reply:
done.